auth/src/main/java/com/firebase/ui/auth/ui/phone/PhoneVerificationActivity.java

```diff
@@ -386,7 +383,7 @@ private void showLoadingDialog(String message) {
 
     private void dismissLoadingDialog() {
         if (mProgressDialog != null) {
-            mProgressDialog.dismiss();
+            mProgressDialog.dismissAllowingStateLoss();
```

Comment:
@samtstern Lol, it was added [way back in `v22.0.0`](https://developer.android.com/reference/android/support/v4/app/DialogFragment.html#dismissAllowingStateLoss()). 😆